File: kotlin-cryptography/src/test/kotlin/me/cryptography/CryptographyTest.kt
package me.cryptography

import org.junit.jupiter.api.Test

class CryptographyTest {
    private val cryptography: Cryptography = Cryptography()

    @Test
    fun testAES256CBCEncryptDecrypt() {
        val text = "Abc1234"
        val password = "MyPassword"

        val encrypted = cryptography.aes256CbcEncrypt(text, password)
        val decrypted = cryptography.aes256CbcDecrypt(encrypted, password)

        println("Encrypted: $encrypted\nDecrypted: $decrypted")
    }
}

File: nodejs-cryptography/test/cryptography.test.js
const { expect } = require('chai')

const cryptography = require('../src/cryptography')

describe('Cryptography', () => {

    describe('testAES256CBCEncryptDecrypt', () => {
        const text = "Abc1234"
        const password = "MyPassword"

        const encrypted = cryptography.aes256CbcEncrypt(text, password)
        const decrypted = cryptography.aes256CbcDecrypt(encrypted, password)

        console.log(`Encrypted: ${encrypted}\nDecrypted: ${decrypted}`)

        expect(decrypted).to.be.equal(text)
    })
})